clients/admin-ui/cypress/e2e/datamap-report.cy.ts

```diff
@@ -354,10 +354,14 @@ describe("Data map report table", () => {
         "deleteCustomReport",
       );
     });
-    it("should show an empty state when no custom reports are available", () => {
-      cy.intercept("GET", "/api/v1/plus/custom-report/minimal*", {
-        fixture: "custom-reports/empty_custom-reports.json",
-      }).as("getEmptyCustomReports");
+    it.only("should show an empty state when no custom reports are available", () => {
+      cy.intercept(
+        "GET",
+        "/api/v1/plus/custom-report/minimal?page=1&size=50&report_type=datamap",
```

Comment:
feeble attempt at preventing this specific test from failing randomly

Reply:
@gilluminate I think you left the .only accidentally